exercises/crypto-square/description.md

```diff
@@ -22,10 +22,16 @@ is normalized to:
 "ifmanwasmeanttostayonthegroundgodwouldhavegivenusroots"
 ```
 
-The plaintext should be organized in to a rectangle.  The size of the
-rectangle (`r x c`) should be decided by the length of the message,
-such that `c >= r` and `c - r <= 1`, where `c` is the number of columns
-and `r` is the number of rows.
+The plaintext should be organized in to a rectangle. The size of the
+rectangle (`r x c`) should be decided by the length of the message.
+
+Find the smallest possible integer `c` such that:
+
+- `r x c >= length(message)`,
+- and `c >= r`,
+- and `c - r <= 1`.
+
+`c` is the number of columns and `r` is the number of rows in the rectangle. 
```

Comment:
Perhaps this line could go  earlier, before the usages of `c` and `r`.
Maybe somewhere around line # 26?